gerror/internal/err_with_custom_convert.go

```diff
@@ -0,0 +1,37 @@
+package internal
+
+import (
+	"fmt"
+
+	"github.com/drshriveer/gtools/gerror"
+)
+
+//go:generate gerror --types ErrWithCustomConvert --skipConvertGen
+
+// ErrWithCustomConvert is just a test.
+//
+//nolint:errname
+type ErrWithCustomConvert struct {
```

Comment:
is this like a test fixture? or is the idea that this can be used by gerror customers?

Reply:
neither? 
This is literally a test of generated code, which is why it's under the internal package. In this file we include the --skipConvertGen flag. (see generate directive: `//go:generate gerror --types ErrWithCustomConvert --skipConvertGen`)

When we run `just generate` the Convert methods should not generate. If they do the tests in this package would break.

Reply:
thats what i meant by fixture**. ok sgtm